### PR TITLE
Add 5.0 deb/rpm migration instructions

### PR DESCRIPTION
**Pull request by jchengsfx**

### CODE REVIEW

File: docs/deb-rpm-repo-migration.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+2. Modify /etc/yum.repos.d/signalfx-agent.repo. Specifically: 
+
+ * Change baseurl to https://splunk.jfrog.io/splunk/signalfx-agent-rpm/release
+
+ * Change gpgkey to https://splunk.jfrog.io/splunk/signalfx-agent-rpm/splunk-B3CD4420.pub
+
+ The file should now look like:
+ ```sh
+ [signalfx-agent]
```
Comment (keitwb) [inline docs/deb-rpm-repo-migration.md]: Leading space?

Comment (keitwb) [reply docs/deb-rpm-repo-migration.md]: What about the equivalent for APT?

Comment (jchengsfx) [reply docs/deb-rpm-repo-migration.md]: > Leading space?

Will remove.

Comment (jchengsfx) [reply docs/deb-rpm-repo-migration.md]: > What about the equivalent for APT?

APT instructions: https://github.com/signalfx/signalfx-agent/pull/1221/files#diff-bfa2b6d288e4320f453c73ac7f5af90cR32

**Pull request merged**